packages/components/src/components/DaScroll.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="scroll__switch"
+    :class="[showed ? 'shown' : '']"
+    @click="scrollTop"
+  >
+    <svgicon name="arrow" class="scroll__switch__icon" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DaScroll",
+
+  data() {
+    return {
+      showed: false,
+    };
+  },
+
+  methods: {
+    scrollTop() {
+      window.scrollTo({ top: 0, behavior: "smooth" });
+    },
+  },
+
+  created() {
+    this.scroll = (event) => {
+      this.showed =
+        document.documentElement.scrollTop >
+        document.documentElement.scrollHeight / 3;
+    };
+    this.scroll = this.scroll.bind(this);
+  },
+
+  mounted() {
+    import("../../icons/arrow");
+    window.addEventListener("scroll", this.scroll);
+  },
+};
+</script>
+
+<style>
+.scroll__switch {
+  height: 48px;
+  width: 48px;
```

Comment:
width and height should be `64px` according to the specs

Reply:
Add progress animation?

Reply:
No need for progress animation just a show and hide animation